mxnet/buildspec.yml

```diff
@@ -1,7 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK mxnet
-version: &VERSION 1.8.0
```

Comment:
for 1.8, file need to be created so someone can use it to build 1.8 on PR. I can add it as well if needed

Reply:
sorry I dont understand what you mean? should we not change the version here?

Reply:
archived buildspec for 1.8, added `buildspec-1-8.yml`